src/js/_store/modules/categories.js

```diff
@@ -11,6 +11,8 @@ export default {
     loading: false,
     doneMessage: '',
     errorMessage: '',
+    deleteCategoryId: null,
+    deleteCategoryName: null,
```

Comment:
[imo]

こういった同じ情報を定義したいときは、オブジェクトにして定義した方が良いかと思います！
他の箇所も同様なので修正してみてください。

```suggestion
    deleteCategory: {
      id: null,
      name: ''
    }
```